src/updateIndex.ts

```diff
@@ -0,0 +1,82 @@
+import * as fs from "fs";
+import * as path from "path";
+import { ProgressLocation, window } from "vscode";
+import { executeJavaLanguageServerCommand } from "./jdtls/commands";
+const axios = require("axios");
```

Comment:
any rocks blocking you from using NodeJS http/https module?

Reply:
will have a try by using NodeJS http/https module

Reply:
not in this PR. please do it later.

Reply:
ok.

Reply:
remove the updating logic, to form a separate PR later.